### PR TITLE
added PR template

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
https://github.com/ergomake/layerform/issues/65

### CODE REVIEW

**Comment by SaumyaBhushan** [comment]
Hi @lucasfcosta . I have added a PR template . Please review .

**Comment by SaumyaBhushan** [comment]
I can add issue template as well . Let me know if you want that

**Review by lucasfcosta — CHANGES_REQUESTED** [review]
Hey @SaumyaBhushan, thanks so much for the PR!

Really appreciate that. Just one comment on my end and then this will be good to go.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Check List (Check all the applicable boxes)  <!-- Follow the above conventions to check the box -->
+
+- [ ] My code follows the code style of this project.
+- [ ] My change requires changes to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] All new and existing tests passed.
+- [ ] This PR does not contain plagiarized content.
+- [ ] The title of my pull request is a short description of the requested changes.
+
```
Comment (vieiralucas) [inline .github/pull_request_template.md:26]: Can you remove this section? Let's keep this as simple as possible for now, we can always add more stuff later if think it's needed.
```suggestion
```

**Review by vieiralucas — CHANGES_REQUESTED** [review]
Hi @SaumyaBhushan 

First of all, thank you so much for taking the time and making this PR.
Overall this is looking pretty good, amazing.

I just think we should remove the `Check List`.
Can you updated it?

Thank you again, cheers.

**Review by lucasfcosta — APPROVED** [review]

**Review by vieiralucas — APPROVED** [review]

**Comment by vieiralucas** [comment]
Thank you @SaumyaBhushan. Welcome the the contributors list 😄 
Closes #65

**Comment by SaumyaBhushan** [comment]
Hi @vieiralucas @lucasfcosta . Please let me know if you have more such issues or issue related to ci-cd Or devops. I would be happy to contribute.

**Pull request merged**